app/models/role.rb

```diff
@@ -36,6 +36,9 @@ def super_associate?; name == :super_associate; end
 	def self.create_for_nonprofit(role_name, email, nonprofit)
 		user = User.find_or_create_with_email(email)
 		role = Role.create(user: user, name: role_name, host: nonprofit)
+
+		MailchimpNonprofitUserAddJob.perform_later(DripEmailList.first, user, nonprofit)
+
 		return role unless role.valid?
```

Comment:
Line 40 should be after the return. We don't want this job being queued unless the role was valid.